### PR TITLE
Zend stack and project sample

### PR DESCRIPTION
**Pull request by kaloyan-raev**
### What does this PR do?

Adds a stack with [Zend Server](http://www.zend.com/en/products/zend_server), based on the [official docker image](https://hub.docker.com/r/_/php-zendserver/). Zend Server is a professional and certified PHP distribution. The docker image includes a free trial Developer license.

Also adds a project sample of the [Astro Splash](https://github.com/kaloyan-raev/AstroSplash) application, which is a demo app based on the [Zend Expressive](https://zendframework.github.io/zend-expressive/) framework.

After creating a workspace with these stack and template the user must:
1. Execute the `configure` command.
2. Execute the `restart zend server` command.
3. Click on the preview link generated by the `restart zend server` command.
### What issues does this PR fix or reference?
- PHP support #2590
### New behavior
- New stack with Zend Server.
- New project sample: Astro Splash demo app, based on Zend Expressive.
### PR type
- [x] Minor change = no change to existing features or docs

Signed-off-by: Kaloyan Raev kaloyan.r@zend.com

### CODE REVIEW

**Comment by TylerJewell** [comment]
@slemeur - ok by me.  I just want to see one text modification made.  Then we can include it into the default stacks configuration.

File: ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
Comment (slemeur) [inline ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg]: @kaloyan-raev : everything looks good, except that you need to add a licence header in the SVG which is compliant with EPL. 

You can check what bitnami did for example: 
https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg

**Review by slemeur — APPROVED** [review]
+1 ! Thanks @kaloyan-raev wonderful to get Zend Stack in Che

File: ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg (hunk 1 above)
Comment (kaloyan-raev) [reply ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg]: Copyright header added.

**Comment by benoitf** [comment]
https://ci.codenvycorp.com/job/che-ci-master/259/org.eclipse.che.core$che-core-ide-stacks/console (there is failure on CI)

**Comment by kaloyan-raev** [comment]
@benoitf It would be great if we have this CI build hooked to PRs, not only to master.

What's wrong with the license header in `type-zend.svg`?

**Comment by benoitf** [comment]
@kaloyan-raev there is a CI PR build https://ci.codenvycorp.com/view/pr-builds/job/che-pullrequests-build/

running `$ mvn clean install` in `ide/che-core-ide-stacks`folder should show the error

**Comment by kaloyan-raev** [comment]
@benoitf Sorry, but I don't find the build for this PR there. Is it possible to configure Jenkins and/or GitHub to have a report on the build result directly in the PR?

The exact error from the build is:

```
[INFO] Checking licenses...
[WARNING] Unknown file extension: /home/kraev/git/che/ide/che-core-ide-stacks/src/main/resources/stacks.json
[WARNING] Missing header in: /home/kraev/git/che/ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg
...
[ERROR] Failed to execute goal com.mycila:license-maven-plugin:2.10:check (check-headers) on project che-core-ide-stacks: Some files do not have the expected license header -> [Help 1]
```

But it is still not clear to me what's wrong with the license header (it is not missing for sure).

**Comment by benoitf** [comment]
@kaloyan-raev Eclipse Che committer has to enter "ci-build" to require a launch of the CI job for the external pull request

`$ mvn com.mycila:license-maven-plugin:2.10:format` is a command to launch formatter

**Comment by kaloyan-raev** [comment]
@benoitf I would be thankful if you simplify the process and launch the CI build automatically for any PR. It would be very helpful for external contributors like me.

I ran the format command. The main thing it does is to change the copyright holder from "Rogue Wave Software, Inc." to "Codenvy, S.A.". This is not the correct thing to be done.

@slemeur asked me to add a header similar to the one in https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg

There the header has "Bitrock Inc." as the copyright holder. I don't see any reason why "Bitrock Inc." is accepted, but not "Rogue Wave Software, Inc.". Perhaps, some build configuration needs to be changed?

**Comment by benoitf** [comment]
@kaloyan-raev this is to avoid to launch jobs that may launch unrelated stuff.
you could submit a PR that create attacks, eat cpu, etc or something else.

looking at the pom.xml of stack there is an exclude
https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/pom.xml#L32

**Comment by kaloyan-raev** [comment]
Thanks. I will push another PR for excluding type-zend.svg.

**Comment by kaloyan-raev** [comment]
@benoitf PR #2703

**Pull request merged**